packages/graphql/src/translate/projection/elements/create-point-element.test.ts

```diff
@@ -58,9 +58,12 @@ describe("createPointElement", () => {
             variable: "this",
         });
 
-        expect(element).toMatchInlineSnapshot(
-            `"point: (CASE WHEN this.point IS NOT NULL THEN { point: this.point, crs: this.point.crs } ELSE NULL END)"`
-        );
+        expect(element).toMatchInlineSnapshot(`
+            "point: (CASE
+                WHEN this.point IS NOT NULL THEN { point: this.point, crs: this.point.crs }
+                ELSE NULL
+            END)"
```

Comment:
Whilst this formatting makes sense when the `CASE` is an individual clause, I think it looks terrible in the context of a projection.

Reply:
mmm, I agree, but it is worth adding logic to change how CASE is formatted?
Between the 2 options, I feel this is the better one, if we were to use it in several places